uploading_packages/remotes.rst

```diff
@@ -55,21 +55,22 @@ To contribute packages to ConanCenter, read the :ref:`ConanCenter guide <conan_c
 conan-center [deprecated]
 -------------------------
 
-**conan-center** was the official repository but is is no longer recommended. It is configured as the second default remote in the Conan
-client to keep backwards compatibility:
+**conan-center** was the official repository but is is **no longer a default remote** in the Conan client and **its usage is completely
```

Comment:
Left this section for reference purposes (as some people might still be using the remote until we completely shut it down), but no objections if we want to remove it, as this is documented in previous versions of the documentation (although maybe not so easy to find it).